managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/OpAddEntry.java

```diff
@@ -165,7 +165,7 @@ public void addComplete(int rc, final LedgerHandle lh, long entryId, Object ctx)
             return;
         }
 
-        if (ledger.getId() != lh.getId()) {
+        if (lh != null && ledger != null && ledger.getId() != lh.getId()) {
```

Comment:
Field `ledgerId` of `LedgerHandle.java` is not wrap class, will not throw NullpointerException.

Mabey `ledger` or `lh` is null ? 

If ledger` is null, that is not a simple null-pointer problem, could you explain the case ?